api-docs/cloud-dedicated/ref.yml

```diff
@@ -1581,24 +1595,29 @@ paths:
       responses:
         "204":
           description: |
-            Success. Data is written and queryable.
+            Success. All points in the batch are written and queryable.
         "400":
           content:
             application/json:
               examples:
-                measurementSchemaFieldTypeConflict:
-                  summary: field type conflict thrown by an explicit database schema
+                partialWriteErrorWithRejectedPoints:
```

Comment:
Can we have 2 examples? One for rejected points error, and another for an entire rejection (e.g. the namespace error).

Reply:
Also, it's configurable. But the error message response you get looks the same.

Suggestions:
* change `partialWriteErrorWithRejectedPoints` to something like `writeErrorWithRejectedPoints`
* If (and only if) you have partial writes enabled, then the non-rejected points are written.

Reply:
Partial-writes is default; Not configurable; no change required.